src/cryptonote_core/service_node_list.cpp

```diff
@@ -706,6 +706,8 @@ namespace service_nodes
 
         info.active_since_height = -info.active_since_height;
         info.last_decommission_height = block_height;
+        //TODO sean
+        info.last_decommission_reason = state_change.reason_consensus_all;
```

Comment:
I wonder if maybe we could pass through the any reasons as well?  Maybe not to display it, but could still be useful to expose it in the RPC.